apps/test-moisture/src/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include "MoistureSensor.h"
+#include "hw_bridge.h"
+#include "mbed.h"
+
+DigitalOut led1(LED1);
+
+int main() {
+  led1 = 0;  // start with LED off
+
+  uint16_t moisture = 0;  // initialize default variables
+  float temperature = 0.0;
+  uint8_t HW_ID     = 0;
+
+  MoistureSensor sensor = MoistureSensor(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
+
+  while (1) {
+    sensor.Reset_Sensor();  // reset all registers on sensor to default values
+
+    ThisThread::sleep_for(500ms);
```

Comment:
wait wheres the using namespace std::chrono....

Reply:
u mean for access to the chrono literals?

Reply:
actually not sure. compiles fine. it might be mbed that adds it implicitly